src/pages/ProductsPage.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div class="q-pa-md">
+    <q-infinite-scroll @load="onLoad" :offset="250">
+      <div v-for="(item, index) in items" :key="index" class="caption">
```

Comment:
It is better to use as key the actual id of the product instead of index (if you have an unique id for objects)